portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/*
+    The function showNext() allows the button to be clicked twice in order to
+    display a personal summary and a photograph. Because the second click is dependent
+    on the first, a global variable is required to determine when to show the picture.
+    Another roundabout solution would be to check whether the 'summaryCointainer'
+    is empty, eliminating the need for a global varible.
+*/
+var summaryVisble = false;
+var pictureVisbile =false;
+function showNext() {
+    if(summaryVisble == false){
+        showSummary();
+        summaryVisble = true;
+    } 
+    else if(pictureVisbile == false){ 
+        showPicture();
+        pictureVisbile = true;
```

Comment:
Another way to avoid using global vars is you can try using appendChild() to add a new node with a special class name to an existing node, and using the hasChildNodes() or getElementsByClassName() to check whether there is a node with a specified class name exist and then decide append a new node or not.